src/analyze/analyze.c

```diff
@@ -82,6 +85,40 @@ static const char *arg_root = NULL;
 STATIC_DESTRUCTOR_REGISTER(arg_dot_from_patterns, strv_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_dot_to_patterns, strv_freep);
 
+#define DOT_COLOR_AFTER     "green"
+#define DOT_COLOR_REQUIRES  "black"
+#define DOT_COLOR_REQUISITE "blue"
+#define DOT_COLOR_WANTS     "grey66"
+#define DOT_COLOR_CONFLICTS "red"
+#define DOT_COLOR_BINDSTO   "magenta"
+#define DOT_COLOR_ONFAILURE "orange"
+#define DOT_COLOR_TRIGGERS  "grey66"
+#define DOT_STYLE_AFTER     "solid"
+#define DOT_STYLE_REQUIRES  "solid"
+#define DOT_STYLE_REQUISITE "solid"
+#define DOT_STYLE_WANTS     "solid"
+#define DOT_STYLE_CONFLICTS "solid"
+#define DOT_STYLE_BINDSTO   "solid"
+#define DOT_STYLE_ONFAILURE "solid"
+#define DOT_STYLE_TRIGGERS  "dotted"
+
+static char *color_after     = NULL;
```

Comment:
please prefix fields that directly map to cmdline arguments with "arg_".

Reply:
dependency_color and dependency_style don't map directly to cmdline arguments, so not prefixing with arg_.